### PR TITLE
Some Starting design for the panel, and few code fix

### PR DESCRIPTION
**Pull request by TheAmazingPeleg**
Some Starting design for the panel, and few code fix

### CODE REVIEW

File: frontend/src/Components/Admin/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function Nav() {
+  return (
+    <div className="admin-nav">
```
Comment (Tamir198) [inline frontend/src/Components/Admin/Nav.jsx:5]: Do you have both nav and admin nav?

File: frontend/src/Components/Admin/Nav.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      <ul>
+        <li><a href="dashboard">Dashboard</a></li>
+        <li><a href="categories">Categories</a></li>
+        <li><a href="addCategory">Add Category</a></li>
+        <li><a href="addProduct">Add Product</a></li>
+        <hr />
```
Comment (Tamir198) [inline frontend/src/Components/Admin/Nav.jsx:11]: Please don't us hr for spacing between elements, use margins and paddings instead

File: frontend/src/index.css [hunk 3]
```diff
@@ -36,14 +36,58 @@ h2 {
...
   color: white;
 }
-
-stat-view {
-  width: 200px;
+.admin-container {
+  width: 100vw;
+  height: 100vh;
+  padding: 0px;
+  padding: 0px;
```
Comment (Tamir198) [inline frontend/src/index.css:43]: Remove one of the padding lines, this is code duplication

File: frontend/src/index.css [hunk 4]
```diff
@@ -36,14 +36,58 @@ h2 {
...
+  list-style-type: none;
+  text-align: center;
+  padding: 0px;
+  margin: 0px;
+}
+.admin-nav li {
+  padding: 10px;
+}
+.admin-nav li:hover {
+  background-color: gray;
```
Comment (Tamir198) [inline frontend/src/index.css:74]: why not use the css colors?

File: frontend/src/Components/Admin/Nav.jsx (hunk 1 above)
Comment (TheAmazingPeleg) [reply frontend/src/Components/Admin/Nav.jsx:5]: Im holding the main nav in the top of the page and the admin nav in the left side of the page (its sidebar nav)

**Review by MasterChiefProject — APPROVED** [review]
please update the team that you created this context

**Pull request merged**